log4j-core/src/main/java/org/apache/logging/log4j/core/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Implementation of Log4j 2.
  */
 @Export
-@Version("2.24.0")
+@Version("2.25.0")
```

Comment:
Version `2.24.0` should be enough for BND.

The previous version was `2.20.2`, but when we make minor changes to a package, we bump the version directly to the next version of Log4j, so it is easier to remember when was a package last modified.